### PR TITLE
Updated Agones and k8s versions.

### PR DESCRIPTION
**Pull request by kemurayama**
Updated following softwares:

- Agones v1.35.0
- Kubernetes:
    - v1.25.15
    - v1.26.10
    - v1.27.7
    - v1.28.3

### CODE REVIEW

**Comment by kemurayama** [comment]
Hi @govargo,

Could you review this PR ?

File: README.md [hunk 1]
```diff
@@ -24,7 +24,6 @@ Install tools in your dev environment:
 - [docker](https://www.docker.com/)
 - [kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#install_kubectl)
 - [helm](https://helm.sh/)
-- [envsubst](https://linux.die.net/man/1/envsubst) (Optional)
```
Comment (govargo) [inline README.md:27]: Do we still use envsubst? If so, we should remain envsubst.
https://github.com/googleforgames/space-agon/blob/main/scripts/setup-skaffold.sh#L46-L47

**Review by govargo — COMMENTED** [review]
I put one comment

File: README.md (hunk 1 above)
Comment (kemurayama) [reply README.md:27]: Thanks. I missed that.

**Comment by kemurayama** [comment]
@govargo Thanks, I added envsubst again.

**Review by govargo — APPROVED** [review]
Thank you. LGTM

**Pull request merged**